.luarc.json

```diff
@@ -1,10 +1,22 @@
 {
-    "diagnostics.globals": [
-        "vim"
-    ],
     "diagnostics.libraryFiles": "Disable",
     "runtime.version": "LuaJIT",
     "workspace.checkThirdParty": "Disable",
-    "workspace.library": ["$PWD/.dependencies"]
+    "workspace.library": [
+        "$PWD/.dependencies",
+        "${3rd}/busted/library",
+        "${3rd}/luv/library",
+        "${3rd}/luassert/library",
```

Comment:
`${3rd}` [will be deprecated in future versions of lua-language-server](https://github.com/LuaLS/lua-language-server/discussions/1950) so I'd like to avoid using it.

Try running `git submodule update --init --recursive` and then changing this file to

```
    "workspace.library": [
        "$PWD/.dependencies/busted/library",
        "$PWD/.dependencies/luassert/library",
        "lua"
    ],
```

That works for me. If it works for you to let's replace it here.